pages/common/gpg2.md

```diff
@@ -0,0 +1,33 @@
+# gpg
+
+> GNU Privacy Guard 2.
+> Homepage: <https://docs.releng.linuxfoundation.org/en/latest/gpg.html>.
+
+- List imported keys:
+
+`gpg2 --list-keys`
+
+- Encrypt doc.txt for alice@example.com (output to doc.txt.gpg):
```

Comment:
Perhaps this will read better?
```suggestion
- Encrypt a specified file with the public key of alice@example.com, writing the output to a new file with `.gpg` appended:
```

Reply:
Thanks! This has been resolved too.